source/_lovelace/conditional.markdown

```diff
@@ -50,6 +54,9 @@ conditions:
     state: "on"
   - entity: switch.decorative_lights
     state_not: "off"
+  - entity: light.ceiling
+    attribute: effect
+    state: sunset
```

Comment:
States must be quoted.

See our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
    state: "sunset"
```

Reply:
Thanks @frenck, fixed